templates/docs/patterns/empty-state.md

```diff
@@ -58,6 +58,10 @@ To import either or all of these components into your project, copy the snippets
 
 // dependencies for user action triggered empty state example
 @include vf-u-vertically-center;
```

Comment:
Thanks for remembering to update this section as well. I think we don't need `vf-u-vertically-center` anymore (class name was removed from the example), and the comment above it is misleading I guess (it's a copy of a comment above). I think it was meant to refer to error management example